tools/debug_tool/templates/pipeline_tree.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+
+<html lang="en">
+<head>
+<meta charset="utf-8" />
+<title>Pipeline Viewer</title>
+<link rel= "stylesheet" type= "text/css" href= "{{ url_for('static',filename='styles.css') }}">
+<script src="https://d3js.org/d3.v6.min.js"></script>
+<script src="http://marvl.infotech.monash.edu/webcola/cola.min.js"></script>
+<script type="text/javascript" src="https://canvasjs.com/assets/script/jquery-1.11.1.min.js"></script>
+<script src="{{url_for('static', filename='pipeline_viewer.js')}}"></script> 
+</head>
+
+<body>
+    <a href="../index.html">cola.js home</a>
+<h1>Kotekan Pipeline</h1>
+<script>
+    get_data().then(function (buffers) {
+        var pipeline = new PipelineViewer(buffers);
+        pipeline.init_svg();
+        pipeline.parse_data();
+        pipeline.create_objs();
+        pipeline.start_buff_ult();
```

Comment:
To simplify this for the user of the `PipelineViewer` object, I would provide just one function call that initializes everything by calling each of these functions internally.